stats.php

```diff
@@ -11,6 +11,10 @@
  * Count publish events regardless of post type
  */
 function track_publish_post( $new_status, $old_status ) {
+	if ( defined( 'WP_IMPORTING' ) && WP_IMPORTING ) {
```

Comment:
Minor: I usually prefer to do strict checks even with booleans (i.e. `true === WP_IMPORTING` ).